src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -98,6 +98,27 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
     setAnchorEl(null);
   };
 
+  const unreviewedActionsDropdown = (
+    <div>
+      <NERButton
+        endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
+        variant="contained"
+        id="unreviewed-cr-actions-dropdown"
+        onClick={handleClick}
+      >
+        Actions
+      </NERButton>
+      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+        <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
+          Review
+        </MenuItem>
+        <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
+          Delete
+        </MenuItem>
```

Comment:
I think these are supposed to have a divider between them like the other two delete dropdowns